### PR TITLE
Group entries for CELL locations

### PR DESCRIPTION
**Pull request by kgugala**
Also some small bugs were fixed

### CODE REVIEW

File: tests/data/golden/modulo6.sdf [hunk 1]
```diff
@@ -5,56 +5,30 @@
...
         (INSTANCE U15)
         (DELAY
             (ABSOLUTE
-                (IOPATH A Z (0.064:0.072:0.072)(0.042:0.082:0.082))
-            )
-        )
-        (DELAY
-            (ABSOLUTE
-                (IOPATH B Z (0.058:0.079:0.079)(0.033:0.073:0.073))
+                    (IOPATH A Z (0.064:0.072:0.072)(0.042:0.082:0.082))
```
Comment (mithro) [inline tests/data/golden/modulo6.sdf]: I this indented too much?

File: tests/data/golden/modulo6.sdf [hunk 2]
```diff
@@ -5,56 +5,30 @@
...
-        )
-        (TIMINGCHECK
-                (SETUP D (posedge CLK) (0.098:0.109:0.109))
-        )
-        (TIMINGCHECK
-                (WIDTH  (negedge CLK) (0.072:0.072:0.072))
-        )
-        (TIMINGCHECK
-                (WIDTH  (posedge S) (0.086:0.086:0.086))
+                    (HOLD D (posedge CLK) (-0.073:-0.019:-0.019))
```
Comment (mithro) [inline tests/data/golden/modulo6.sdf]: Are these indented too much?

File: tests/data/golden/modulo6.sdf (hunk 1 above)
Comment (kgugala) [reply tests/data/golden/modulo6.sdf]: fixed

File: tests/data/golden/modulo6.sdf (hunk 2 above)
Comment (kgugala) [reply tests/data/golden/modulo6.sdf]: fixed

**Pull request merged**